tools/test-proxy/documentation/methodology.md

```diff
@@ -0,0 +1,5 @@
+# Test-Proxy Introduction
+
+TODO: create 10k foot intro and detail the process for beginning
```

Comment:
I assume you plan to fill in the various TODO's later?

Reply:
Yep.

Reply:
I just ripped the file out. No reason to pollute the repo until it's done.